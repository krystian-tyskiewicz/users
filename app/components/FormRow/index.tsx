'use client'

import styled from 'styled-components'

const FormRow = styled.div`
  position: relative;
`

export default FormRow
