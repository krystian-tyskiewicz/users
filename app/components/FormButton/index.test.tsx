import { render } from '@testing-library/react'
import FormRow from '.'

jest.mock('next/navigation', () => ({
  useRouter: jest.fn(),
}))

it('renders correctly', () => {
  const { container } = render(<FormRow>Test</FormRow>)
  expect(container).toMatchSnapshot()
})
