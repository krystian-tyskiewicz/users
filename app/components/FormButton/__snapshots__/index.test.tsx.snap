// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <button
    class="sc-bdvumH haoqpZ sc-gsDKeI iIPYjV"
    type="button"
  >
    Test
  </button>
</div>
`;
